swagger-spec/search/registrations.yaml

```diff
@@ -1 +1,159 @@
 # /search/registrations/
+get:
+  summary: Search the OSF for registrations
+  description: >-
+    This endpoint returns registrations that have been found by the given Elasticsearch query. All registrations
+    returned are serialized a registrations.
```

Comment:
Same typo - Either 'are serialized as registrations' or 'are serialized registrations' or 'are serialized' should be fine.